src/xtopcom/xtxexecutor/src/xtransaction_context.cpp

```diff
@@ -285,11 +286,37 @@ int32_t xtransaction_transfer::source_fee_exec() {
     auto transfer_amount = get_amount();
     // no check transfer amount for genesis state
     if (!is_contract_address(common::xaccount_address_t{ m_trans->get_source_addr() }) && transfer_amount) {
-        ret = m_fee.update_tgas_disk_sender(transfer_amount, false);
+        auto fork_config = top::chain_upgrade::xtop_chain_fork_config_center::chain_fork_config();
+        auto clock = m_account_ctx->get_timer_height();
+        xdbg("wish account context timer:%llu, timestamp(second):%llu, forkclock:%llu", clock, m_trans->get_transaction()->get_fire_timestamp(), chain_upgrade::unit_tx_opt_clock);
+        if (m_trans->get_transaction()->get_fire_timestamp() >= chain_upgrade::unit_tx_opt_clock * 10) {
```

Comment:
chain_upgrade::unit_tx_opt_clock * 10  change to   fork_config.block_unit_tx_opt_fork_point * 10